client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	'IVF treatment': i18n.translate( 'IVF treatment' ),
+	Ideas: i18n.translate( 'Ideas' ),
+	Illustration: i18n.translate( 'Illustration' ),
+	'Image editing': i18n.translate( 'Image editing' ),
+	'Importing and exporting': i18n.translate( 'Importing and exporting' ),
+	Information: i18n.translate( 'Information' ),
+	'Information technology': i18n.translate( 'Information technology' ),
+	Inspiration: i18n.translate( 'Inspiration' ),
+	Insurance: i18n.translate( 'Insurance' ),
+	'Intellectual property lawyer': i18n.translate( 'Intellectual property lawyer' ),
+	'Interior design': i18n.translate( 'Interior design' ),
+	'International logistics': i18n.translate( 'International logistics' ),
+	'Internet provider': i18n.translate( 'Internet provider' ),
+	Investing: i18n.translate( 'Investing' ),
+	'Investment consultancy': i18n.translate( 'Investment consultancy' ),
+	Islam: i18n.translate( 'Islam' ),
+	Issues: i18n.translate( 'Issues' ),
+	'Italian restaurant': i18n.translate( 'Italian restaurant' ),
+	'Japanese cartoons': i18n.translate( 'Japanese cartoons' ),
+	Jesus: i18n.translate( 'Jesus' ),
+	Jewelry: i18n.translate( 'Jewelry' ),
+	'Job consultancy': i18n.translate( 'Job consultancy' ),
+	'Job listing': i18n.translate( 'Job listing' ),
+	'Job posting': i18n.translate( 'Job posting' ),
+	Jobs: i18n.translate( 'Jobs' ),
+	Jokes: i18n.translate( 'Jokes' ),
+	Journalisim: i18n.translate( 'Journalisim' ),
+	Journalism: i18n.translate( 'Journalism' ),
+	Juicebar: i18n.translate( 'Juicebar' ),
+	Kids: i18n.translate( 'Kids' ),
+	'Kids yoga': i18n.translate( 'Kids yoga' ),
+	Kitchen: i18n.translate( 'Kitchen' ),
+	Knowledge: i18n.translate( 'Knowledge' ),
+	LARP: i18n.translate( 'LARP' ),
+	Law: i18n.translate( 'Law' ),
+	Leadership: i18n.translate( 'Leadership' ),
+	Leather: i18n.translate( 'Leather' ),
+	Lemons: i18n.translate( 'Lemons' ),
+	Life: i18n.translate( 'Life' ),
+	'Life coach': i18n.translate( 'Life coach' ),
+	'Life inspiration': i18n.translate( 'Life inspiration' ),
+	Lifestyle: i18n.translate( 'Lifestyle' ),
+	'Lifestyle blog': i18n.translate( 'Lifestyle blog' ),
+	'Literary portfolio': i18n.translate( 'Literary portfolio' ),
+	Literature: i18n.translate( 'Literature' ),
+	Living: i18n.translate( 'Living' ),
+	'Local music': i18n.translate( 'Local music' ),
+	'Local news': i18n.translate( 'Local news' ),
+	Love: i18n.translate( 'Love' ),
+	'Love stories': i18n.translate( 'Love stories' ),
+	'Maid services': i18n.translate( 'Maid services' ),
+	Makeup: i18n.translate( 'Makeup' ),
+	Manga: i18n.translate( 'Manga' ),
+	Maps: i18n.translate( 'Maps' ),
+	Marketing: i18n.translate( 'Marketing' ),
+	'Marketing services': i18n.translate( 'Marketing services' ),
+	Massages: i18n.translate( 'Massages' ),
+	Mathematics: i18n.translate( 'Mathematics' ),
+	Mechanic: i18n.translate( 'Mechanic' ),
+	'Medial news': i18n.translate( 'Medial news' ),
+	Medical: i18n.translate( 'Medical' ),
+	'Medical technology': i18n.translate( 'Medical technology' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **40** times:
`translate( 'Technology' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).